src/msw/window.cpp

```diff
@@ -236,6 +236,27 @@ class GestureFuncs
         return ms_pfnSetGestureConfig;
     }
 
+    typedef BOOL (WINAPI *RegisterTouchWindow_t)(HWND,ULONG);
```

Comment:
Actually we shouldn't need to load these functions dynamically any longer, they're all available since Windows 7 and we don't support anything earlier (or even it) any longer. If you prefer, we can keep this code as is for this PR, but we could (and should) just remove all this dynamic loading stuff later.